docs/articles/intro.qmd

```diff
@@ -5,10 +5,14 @@ aliases:
 html-table-processing: none
 ---
 
-# Great Tables
+<div style="text-align: center;">
+
+<a href="https://posit-dev.github.io/great-tables/"><img src="https://raw.githubusercontent.com/posit-dev/great_tables/main/images/GT_logo.svg" width="350px"/></a>
```

Comment:
Is there a quarto friendly way to do this? This links to the same image no matter what branch you're on